lib/bot.js

```diff
@@ -52,7 +52,13 @@ Bot.prototype.connect = function() {
     channels: this.channels,
     floodProtection: true,
     floodProtectionDelay: 500
-  }, this.ircOptions);
+  }, this.options.ircOptions);
+
+  var slackOptions = _.assign({
```

Comment:
You're not actually using the `slackOptions` variable for anything here. If you want to assign defaults you can do that in the constructor, with `this.options.slackOptions` instead. Example: https://github.com/robertkety/slack-irc/blob/join-part-quit/lib/bot.js#L30 You can still use the assign style you're doing here though :)